clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5192,6 +5227,18 @@ static void handleIntelFPGAPumpAttr(Sema &S, Decl *D, const ParsedAttr &Attr) {
     D->addAttr(IntelFPGAMemoryAttr::CreateImplicit(
         S.Context, IntelFPGAMemoryAttr::Default));
 
+  if (Attr.getKind() == ParsedAttr::AT_IntelFPGADoublePump &&
+      checkDeprecatedFPGASYCLSpelling(S, Attr)) {
+    S.Diag(Attr.getLoc(), diag::note_spelling_suggestion)
+        << "'intel::doublepump'";
+    return;
+  } else if (Attr.getKind() == ParsedAttr::AT_IntelFPGASinglePump &&
```

Comment:
Please don't do else after return.
```suggestion
    return;
  }
if (Attr.getKind() == ParsedAttr::AT_IntelFPGASinglePump &&
```

Reply:
Done